roles/common/tasks/secrets.yml

```diff
@@ -0,0 +1,14 @@
+---
+# tasks file for common secrets
+- name: Generate tracking/search shared secreti (none provided)
+  set_fact:
+    reposearch_shared_secret: "{{ lookup('password', '/dev/null') }}"
+  delegate_to: localhost
+  delegate_facts: true
+  run_once: true
```

Comment:
is not really required anymore?

Reply:
removing it wouldn't break things but we would iterate against the hosts list (the hosts the repo and search roles applies to) and voerride the generated secret everytime. So that's still usefull